docs/blog/posts/why_mesop.md

```diff
@@ -0,0 +1,53 @@
+---
+date: 2024-05-13
+---
+
+# Why Mesop? Or, Why Yet Another Python UI Framework?
+
+Choosing a Python UI framework often feels like a compromise. Many frameworks are great for demos but difficult to customize and polish. In contrast, more powerful JavaScript frameworks can be unapproachable for developers without a front-end background. Mesop offers a different path, providing a framework that aims to be both intuitive and scalable. It allows you to build anything from quick prototypes to robust applications, all without leaving the comfort of writing Python.
+
+I want to share a couple concrete ways in which Mesop achieves this.
+
+## Build UIs with Functions (i.e. Components)
+
+Mesop embraces a component-based philosophy where the entire UI is composed of reusable, building blocks which are called components. Using a component is as simple as calling a Python function. This approach offers several benefits:
```

Comment:
Nice this section is strong. Clear and I think gives some compelling reasons. Good usage of bullets.